### ISSUE TITLE
node v20

### ISSUE THREAD

**Issue by ahathaway** [ChristopheCVB/strapi-plugin-soft-delete#13]
Please add support for node v20 as that is LTS and also preferred by current strapi 4 releases 

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'strapi-plugin-soft-delete@0.9.5',
npm WARN EBADENGINE   required: { node: '>=14.19.1 <=18.x.x', npm: '>=6.0.0' },
npm WARN EBADENGINE   current: { node: 'v20.11.0', npm: '10.2.4' }
npm WARN EBADENGINE }
```

**Comment by ChristopheCVB** [comment]
Hi there!
Indeed, node 20 should be supported without any problem, feel free to open a PR

**Comment by petross3530mx** [comment]
> Hi there! Indeed, node 20 should be supported without any problem, feel free to open a PR

Hi @ChristopheCVB , I noticed that latest master includes requested updates, please, can you bring update to be available also from npm? There is still 0.9.5 with node <18 version support

**Comment by ChristopheCVB** [comment]
@petross3530mx Not sure what I did wrong but I published a new version anyway with latest changes
